### PR TITLE
Remove useless comment

### PR DESCRIPTION
**Pull request by ottaviano**
https://twitter.com/Stof70/status/1019515955083403264  🎉 🎉 🎉

### CODE REVIEW

File: src/Admin/Extension/AlgoliaIndexedEntityAdminExtension.php [hunk 1]
```diff
@@ -17,7 +17,6 @@ public function __construct(AlgoliaIndexedEntityManager $manager)
     }
 
     /**
-     * @param AdminInterface                $admin
      * @param AlgoliaIndexedEntityInterface $object
```
Comment (HeahDude) [inline src/Admin/Extension/AlgoliaIndexedEntityAdminExtension.php]: Should be removed and added in the signature (same for other occurrences)

File: src/CitizenProject/CitizenProjectFactory.php [hunk 2]
```diff
@@ -91,8 +87,6 @@ public function createFromCitizenProjectCreationCommand(CitizenProjectCreationCo
      * The format must be something like "33 0102030405"
      *
      * @param string $phoneNumber
```
Comment (HeahDude) [inline src/CitizenProject/CitizenProjectFactory.php]: Should be type hinted in the signature instead.

File: src/Entity/CitizenProject.php [hunk 3]
```diff
@@ -507,7 +507,6 @@ public static function createForAdherent(
     /**
      * Marks this citizen project as approved.
      *
-     * @param string $timestamp
      *
```
Comment (HeahDude) [inline src/Entity/CitizenProject.php]: this empty line can be removed too (should be fixed in the fixer btw)

File: src/CitizenProject/CitizenProjectFactory.php (hunk 2 above)
Comment (HeahDude) [reply src/CitizenProject/CitizenProjectFactory.php]: (many occurrences)

**Comment by thomasbisignani** [comment]
:1st_place_medal:

**Pull request merged**